model/special_clause.py

```
@@ -824,26 +820,26 @@ def test_clause_system():
     finalize_result = finalize_room_session_spring(room_id, "max_rounds")
     
     if finalize_result["success"]:
-        print(f"✅ 세션 종료 완료")
+        print(f"OK 세션 종료 완료")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)